examples/Cargo.toml

```diff
@@ -1,18 +1,21 @@
 [package]
 name = "odra-examples"
-version = "0.8.0"
+version = "0.8.1"
 edition = "2021"
 
 [dependencies]
-odra = { path = "../odra", default-features = false }
-odra-modules = { path = "../modules", default-features = false }
+odra = { path = "../odra", version = "0.8.1", default-features = false }
```

Comment:
Specifying `version` is not needed here. Examples are never published to crates.io